packages/mui-system/src/InitColorSchemeScript/InitColorSchemeScript.tsx

```diff
@@ -8,6 +8,11 @@ export const DEFAULT_COLOR_SCHEME_STORAGE_KEY = 'color-scheme';
 export const DEFAULT_ATTRIBUTE = 'data-color-scheme';
 
 export interface InitColorSchemeScriptProps {
+  /**
+   * The default mode when the storage is empty (user's first visit)
+   * @default 'system'
+   */
+  defaultMode?: 'system' | 'light' | 'dark';
```

Comment:
I'm not sure about the prop name: "default"? It sounds more like "initial" as there is a notion of session.